crates/typst/src/geom/size.rs

```diff
@@ -18,6 +18,11 @@ impl Size {
     pub fn to_point(self) -> Point {
         Point::new(self.x, self.y)
     }
+
+    /// Convert to a ratio.
```

Comment:
The doc comment could be clearer since there isn't a natural conversion from size to ratio.